standalone-metastore/metastore-common/src/gen/thrift/gen-javabean/org/apache/hadoop/hive/metastore/api/AlterPartitionsRequest.java

```diff
@@ -1028,14 +1028,14 @@ public void read(org.apache.thrift.protocol.TProtocol iprot, AlterPartitionsRequ
           case 4: // PARTITIONS
             if (schemeField.type == org.apache.thrift.protocol.TType.LIST) {
               {
-                org.apache.thrift.protocol.TList _list1360 = iprot.readListBegin();
-                struct.partitions = new java.util.ArrayList<Partition>(_list1360.size);
-                @org.apache.thrift.annotation.Nullable Partition _elem1361;
-                for (int _i1362 = 0; _i1362 < _list1360.size; ++_i1362)
+                org.apache.thrift.protocol.TList _list1376 = iprot.readListBegin();
```

Comment:
we don't make any changes on the AlterPartitionsRequest, not sure how these comes from?

Reply:
Thrift file generation is little complex to understand. I believe it has an order for the fields in the thrift objects. if we insert a new field in one of the thrift objects, it re-generates all other thrift objects also.